homeassistant/components/light/mqtt_json.py

```diff
@@ -108,6 +127,11 @@ def __init__(self, hass, name, topic, qos, retain,
 
         self._flash_times = flash_times
 
+        self._templates = {}
+        for key, tpl in list(templates.items()):
```

Comment:
Is it not possible to omit the `list()` here? The `for` loop should go over the generator